person.rb

```diff
@@ -11,6 +11,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ] Kindly create a method that creates a new rental object of class Rental and passes `date`, `book`, and `person` itself as a parameter to achieve the `has-many` association. You may learn more about both ways association from [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships).